admin_manual/installation/server_tuning.rst

```diff
@@ -147,6 +150,9 @@ Nextcloud to use Imaginary by editing your `config.php`:
     ],
     'preview_imaginary_url' => 'http://<url of imaginary>',
 
+.. warning::
+
+   Make sure to start Imaginary with the `-return-size` command line parameter. Otherwise, there will be a minor performance impact. The flag requires a recent version of Imaginary.
```

Comment:
```suggestion
   Make sure to start Imaginary with the `-return-size` command line parameter. Otherwise, there will be a minor performance impact. The flag requires a recent version of Imaginary and is by default added to the `aio-imaginary` container.
```